py_mmd_tools/nc_to_mmd.py

```diff
@@ -615,11 +616,24 @@ def to_mmd(self, *args, **kwargs):
         # Data access should not be read from the netCDF-CF file
         _ = mmd_yaml.pop('data_access')
         # Add OPeNDAP data_access if "netcdf_product" is OPeNDAP url
+        rm_file_for_checksum_calculation = False
         if 'dodsC' in self.netcdf_product:
             self.metadata['data_access'] = self.get_data_access_dict(ncin, **kwargs)
+            if netcdf_local_path:
+                file_for_checksum_calculation = netcdf_local_path
+            else:
+                resource = ''
+                for access in self.metadata['data_access']:
+                    if access['type'] == 'HTTP':
+                        resource = access['resource']
+                print('Downloading NetCDF file to calculate checksum...')
+                file_for_checksum_calculation = wget.download(resource)
+                rm_file_for_checksum_calculation = True
+            file_size = pathlib.Path(file_for_checksum_calculation).stat().st_size / (1024 * 1024)
         else:
             self.metadata['data_access'] = []
             file_for_checksum_calculation = self.netcdf_product
+            file_size = pathlib.Path(self.netcdf_product).stat().st_size / (1024 * 1024)
```

Comment:
file_size can be outside the if else using the file_for_checksum_calculation

Reply:
yes - done now (without local testing..)